doc/distro.md

```diff
@@ -0,0 +1,29 @@
+## How to set up kube-spawn on various Linux distros
+
+### Fedora
+
+Fedora 26 or newer is needed, as systemd v233 or newer is necessary.
+
+Other than that, normally kube-spawn should work fine on Fedora.
```

Comment:
Don't we need to explain how to disable SELinux? https://github.com/kinvolk/kube-spawn/issues/77#issuecomment-322189571

And about firewalld, is it a concern or does it need to be disabled as well?

Reply:
@alban Yes, we should probably explain about disabling SELinux. I'm not sure if that belongs to distro-specific issues, so I'll need to create a section about distro-wide issues.

We don't need to care about firewalld, as kube-spawn already adds iptables rules automatically.

Reply:
@alban Updated the PR.
Added more dependencies like btrfs-progs, libselinux-utils, polkit, and removed unnecessary dependencies from Vagrantfile.
In the document, I added a section `Common`, which explains about non-distro-specific requirements. There you can find a description about SELinux.